src/sentry/api/validators/project_codeowners.py

```diff
@@ -23,12 +27,11 @@ def validate_association_emails(
 
 
 def validate_association_actors(
-    raw_items: Collection[str],
-    associations: Iterable[ExternalActor],
+    raw_items: list[str],
```

Comment:
The recommendation is to use `Sequence` on the input parameters (less specific) and `list` on the return parameters (more specific).

https://www.notion.so/sentry/sentry-typing-guidelines-best-practices-471d5aacf262476188982ce70f909e00#471d5aacf262476188982ce70f909e00